myapp/app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,75 @@
+class Admin::UsersController < ApplicationController
+  before_action :require_admin
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @users = User.active
+  end
+
+  def show
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:notice] = I18n.t 'msg_account_create_success'
+      redirect_to admin_user_path(@user)
+    else
+      flash.now[:alert] = I18n.t 'msg_account_create_failure'
+      render :new, status: 422
+    end
+  end
+
+  def edit
+  end
+
+  def update
```

Comment:
The last admin can disable their own admin flag, potentially leaving no admins. Is there a safeguard in place?